docs/ops/sort/TopK_1.md

```diff
@@ -51,15 +51,15 @@
 
 **Detailed Description**
 
-Output tensor is populated by values computes in the following way:
+The output tensor is populated by values computed in the following way:
 
     output[i1, ..., i(axis-1), j, i(axis+1) ..., iN] = top_k(input[i1, ...., i(axis-1), :, i(axis+1), ..., iN]), k, sort, mode)
 
 So for each slice `input[i1, ...., i(axis-1), :, i(axis+1), ..., iN]` which represents 1D array, top_k value is computed individually.
 
 Sorting and minimum/maximum are controlled by `sort` and `mode` attributes:
   * *mode*=`max`, *sort*=`value` - descending by value
-  * *mode*=`max`, *sort*=`index` - ascending by index
+  * *mode*=`max`, *sort*=`index` - descending by index
```

Comment:
@tomdol Hi Tomasz, could you please elaborate the reason why we want descending sort for sort=index now, even for mode=max? Because if this is the behavior we want for TopK_1, as well as TopK_3 and TopK_11, then I think all the implementation in ref topk and plugin topk should revise accordingly.
cc @dmitry-gorokhov

Reply:
I've only updated the spec because I think the previous statement was incorrect and could have been a copy-and-paste error. The `mode` attribute determines the order of elements while `sort` decides whether the sorting should be done by comparing the indices or by comparing the values. I could revert those changes in the spec if the previous version was aligned with the code.

Reply:
Thanks Tomasz for the prompt reply. I think the original spec, always applying ascend sorting for sort=index, is reasonable. And yes, they are aligned with the code. Please revert those changes. Thanks a lot!

cc @dmitry-gorokhov @rkazants